storages/merger/bucket_meta_data.go

```diff
@@ -17,17 +22,22 @@ func MergePartially(firstResponse backend.Response, successes []backend.Response
 	if err != nil {
 		log.Printf("Could not read first response body reqID %s, reason %s", firstResponse.ReqID(), err)
```

Comment:
Shouldn't there be a `return`?

Reply:
No, we want to return any response we have